rosdep/base.yaml

```diff
@@ -3665,6 +3665,19 @@ openssl:
   fedora: [openssl]
   gentoo: [dev-libs/openssl]
   ubuntu: [openssl]
+optipng:
+  debian:
+    buster: [optipng]
+    jessie: [optipng]
+    sid: [optipng]
+    stretch: [optipng]
+    wheezy: [optipng]
+  fedora: [optipng]
+  gentoo: [optipng]
+  ubuntu:
```

Comment:
Also all the same so should be collapsed, especially since this listing doesn't cover all our supported distros.